src/content/insertable-streams/video-crop/js/worker.js

```diff
@@ -0,0 +1,35 @@
+/*
+ *  Copyright (c) 2021 The WebRTC project authors. All Rights Reserved.
+ *
+ *  Use of this source code is governed by a BSD-style license
+ *  that can be found in the LICENSE file in the root of the source
+ *  tree.
+ */
+
+'use strict';
+
+const canvas = new OffscreenCanvas(640, 360);
+const ctx = canvas.getContext('2d', {desynchronized: true});
+
+async function transform(frame, controller) {
+  // Cropping is a bit too complex. See https://github.com/w3c/webcodecs/issues/281
+  ctx.drawImage(frame,
+      320, 180,
+      640, 360,
+      0, 0, 640, 360);
```

Comment:
nit: scale params aren't really needed if you're not scaling

Reply:
oddly omitting them led to a overload resolution error...

Reply:
Sorry, my mistake; the 5-parameter version does scaling, not cropping.

Reply:
the overload behaviour is clearly a mess hence the doc link :-)